setup.py

```diff
@@ -17,28 +17,42 @@
 
 scripts = ['tx']
 
-install_requires = []
+install_requires = ['pyinstaller']
 extra_args = {}
 import platform
 if platform.system() == 'Windows':
```

Comment:
Just seeing this message now! woa.
Okay, got it.

> On Oct 17, 2015, at 06:21, Diego Búrigo Zacarão <notifications@github.com> wrote:
> 
> In setup.py <https://github.com/transifex/transifex-client/pull/99#discussion_r42300203>:
> 
> > @@ -17,28 +17,42 @@
> >  
> >  scripts = ['tx']
> >  
> > -install_requires = []
> > +install_requires = ['pyinstaller']
> >  extra_args = {}
> >  import platform
> >  if platform.system() == 'Windows':
> Just want to make it clear. We should just get rid of this if platform.system() == 'Windows' and all content under it. We just need the tx.spec working fine when issuing.
> 
> I believe generating the tx.exe in pip install is a bad idea because we won't be able to uninstall it on `pip uninstall.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/transifex/transifex-client/pull/99/files#r42300203>.
>